loadtest.go

```diff
@@ -79,6 +79,9 @@ func (l *loadTest) runUser() {
 
 	// run the enabled tasks until the loadTest is stopped
 	errG, ctx := errgroup.WithContext(ctx)
+	if reconnectInterval := l.w.Conf().Subscriber.ReconnectInterval; reconnectInterval > 0 {
+		errG.Go(func() error { return l.cycleConnection(ctx, client, reconnectInterval) })
```

Comment:
I wonder if it's easier to just make the reconnect interval a timeout on the context, and so when the interval is reached, `runUser` will return and boomer will spawn a new user? This feels more representative of what we are trying to achieve (i.e. users coming and going), and avoids adding `Connect` to the interface

Reply:
will boomer start a new user on the old one terminating cleanly? I can't see that the task Fn gets run in a for loop

Reply:
It will yes. Here's where boomer runs the task function in a loop:

https://github.com/myzhan/boomer/blob/master/runner.go#L134-L150